chia/simulator/SimulatorFullNodeRpcClient.py

```diff
@@ -0,0 +1,21 @@
+from chia.rpc.full_node_rpc_client import FullNodeRpcClient
+from chia.types.blockchain_format.sized_bytes import bytes32
+from chia.util.bech32m import encode_puzzle_hash
+
+
+class SimulatorFullNodeRpcClient(FullNodeRpcClient):
+    async def farm_block(self, target_ph: bytes32, number_of_blocks: int = 1) -> None:
+        address = encode_puzzle_hash(target_ph, "txch")
+        await self.fetch("farm_block", {"address": address, "blocks": number_of_blocks})
+
+    async def farm_transaction_block(self, target_ph: bytes32, number_of_blocks: int = 1) -> None:
+        address = encode_puzzle_hash(target_ph, "txch")
+        await self.fetch("farm_block", {"address": address, "blocks": number_of_blocks, "guarantee_tx_block": True})
+
+    async def enable_auto_farming(self, enable_auto_farming: bool) -> bool:
```

Comment:
We should probable rename to `set_auto_farming`, since if we turn it off, the word enable doesn't make sense.